src/Sulu/Bundle/ContactBundle/Api/Contact.php

```diff
@@ -245,6 +245,26 @@ public function getPosition()
         return $position->getId();
     }
 
+    /**
+     * Get position.
+     *
+     * @return string
+     *
+     * @VirtualProperty
+     * @SerializedName("positionName")
+     * @Groups({"contactPosition"})
```

Comment:
I think I would make that part of the `fullContact` group instead.

Reply:
@danrot then we should also join the position else it will get lazy loaded

Reply:
That's right, thought that was already joined, but it seems not the `co_position` table, which finally holds the name...